packages/0x.js/src/contract_wrappers/ether_token_wrapper.ts

```diff
@@ -163,6 +163,15 @@ export class EtherTokenWrapper extends ContractWrapper {
     public unsubscribeAll(): void {
         super.unsubscribeAll();
     }
+    /**
+     * Retrieves the Ethereum address of the EtherToken contract deployed on the network
+     * that the user-passed web3 provider is connected to. If it's an unknown private network - undefined is returned.
+     * @returns The Ethereum address of the EtherToken contract or undefined.
+     */
+    public getContractAddressIfExists(): string | undefined {
+        const contractAddress = artifacts.EtherTokenArtifact.networks[this._networkId].address;
```

Comment:
Won't `artifacts.EtherTokenArtifact.networks[this._networkId]` return undefined if the `networkId` is not in the `networks` object? Then we are looking for `address` property on undefined and this will throw. This needs to be fixed. Let's add a regression test for this.

Reply:
Was this addressed?